### PR TITLE
Eng 10399 more info in conflict log

### PR DESCRIPTION
**Pull request by luyangco**

### CODE REVIEW

File: src/ee/common/tabletuple.h [hunk 1]
```diff
@@ -386,6 +386,20 @@ class TableTuple {
         return std::string(retval, 0, retval.length() - 1);
     }
 
+    std::string toJsonArray(const std::vector<std::string>& columnNames) const {
+        int totalColumns = sizeInValues();
+        Json::Value array(Json::arrayValue);
+
+        array.resize(totalColumns);
+        for (int i = 0; i < totalColumns; i++) {
+            array[i] = columnNames[i] + ":" + getNValue(i).toString();
```
Comment (nshi) [inline src/ee/common/tabletuple.h]: Is this still valid JSON, by creating an array of key:value pairs?

File: src/ee/storage/BinaryLogSink.cpp [hunk 2]
```diff
@@ -156,13 +162,22 @@ void setConflictOutcome(boost::shared_ptr<TempTable> metadataTable, bool acceptR
```
Comment (nshi) [inline src/ee/storage/BinaryLogSink.cpp]: |`line 231`|
|---|
I think you can get this directly by calling UniqueId::timestampSinceUnixEpoch().

**Comment by nshi** [comment]
|`src/ee/storage/BinaryLogSink.cpp:351`|
|---|
This should also be protected by the if.

File: src/ee/common/tabletuple.h (hunk 1 above)
Comment (luyangco) [inline src/ee/common/tabletuple.h]: I think this is legit, it is still an JSON array with colon inside the value of each element, JSON spec doesn't require to escape any colon in value section.

Comment (luyangco) [inline src/ee/common/tabletuple.h]: I mean, this is still an array of strings, not an array of key:value pairs

Comment (nshi) [inline src/ee/common/tabletuple.h]: I'm thinking that maybe a JSON dictionary makes more sense here, e.g. "{c1: val1, c2: val2}".

**Pull request merged**